### DISCUSSION TITLE
Can signac software be used for the processed scATAC data without fragments.tsv.gz or singlecell.csv

### DISCUSSION THREAD

**Discussion by sunshine1126** [stuart-lab/signac#906] [Q&A] [answered]
Hello, thank you for this awesome tool for scATAC-seq analysis!
I got processed files of scATAC-seq data but it lacks two files (fragments.tsv.gz and singlecell.csv).  Through consulting the information, I knew that the singlecell.csv file may affect the filtering of data. I don't know whether the  fragments.tsv.gz file has a great impact on data analysis. That is, I don't know whether I can continue to analyze it using the signac software later. 
![image](https://user-images.githubusercontent.com/32659317/145355823-8566422a-5555-4d64-88d4-2019e2173d3d.png)

**Comment by timoast** [comment, marked as answer]
You can run an analysis using only a count matrix, but many functions in Signac will not be able to be run without a fragment file (eg, CoveragePlot, TSSEnrichment, NucleosomeSignal, Footprint, FeatureMatrix, GeneActivity...)

**Reply by sunshine1126** [reply]
@timoast Thanks for your reply, and I will try it.
